src/Components/loadingScreen.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from "react";
```

Comment:
@lambert3 , this should be in a separate branch. If this PR is related to temperate variation charts, then the loading feature Paul was talking about should be in a new branch created off of master

Reply:
thanks @RonKbS , let me implement this right away